apidocs/overview.md

```diff
@@ -0,0 +1,438 @@
+# Authentication
+
+All REST API resources are authenticated with [personal access tokens](https://docs.launchdarkly.com/v2.0/docs/api-access-tokens) or session cookies. Other authentication mechanisms are not supported. You can manage personal access tokens on your [Account settings](https://app.launchdarkly.com/settings/tokens) page. 
+
+LaunchDarkly also has SDK keys, mobile keys, and client-side IDs that are used by our server-side SDKs,  mobile SDKs, and client-side JavaScript SDKs, respectively. **These keys cannot be used to access our REST API**. These keys are environment-specific, and can only perform read-only operations (fetching feature flag settings).
+
+| Auth mechanism | Allowed resources | Use cases |
```

Comment:
Do we want to use Markdown or HTML tables in these docs?

Reply:
Great question. I guess it depends on how we want them styled - if the default markdown styling is enough we can probably get away with md, but I suspect we want to style them. We have other styling debt that we need to address (e.g. the callouts definitely don't look as good), but it might be worth addressing that in a subsequent PR.